modules/client-configuration/pages/client-proxy-script.adoc

```diff
@@ -0,0 +1,69 @@
+[[script-client-proxy]]
+= Register Clients  to a Proxy with a Bootstrap Script
+
+
+You can register clients through the {productname} Proxy with a bootstrap script.
+This is done almost the same way as registering clients directly with the {productname} Server.
+
+You create the bootstrap script in the {productname} Server container with the [command]``mgr-bootstrap`` command line tool.
+[command]``mgr-bootstrap`` requires some parameters such as the fully qualified domain name (FQDN) of the proxy, activation keys, or GPG keys.
+These parameters depend on your specific setup, but the FQDN of the proxy is mandatory.
+
+The bootstrap script then deploys all necessary information to the clients.
+
+
+
+.Procedure: Registering Clients to a Proxy with a Bootstrap Script
+
+. Create a client activation key on the {productname} Server using the {webui}.
+  For more information, see xref:client-configuration:activation-keys.adoc[].
+. On the container host of the server, start a terminial in the container:
+
++
+
+[source,shell]
+----
+mgrctl term
+----
+
++
+
+--
+.. In the container, to create the bootstrap script, execute the [command]``mgr-bootstrap`` command, for example, with these options:
+
++
+
+[source,shell]
+----
+mgr-bootstrap --hostname=MLM_PROXY --activation-keys=ACTIVATION_KEY \
+  --script bootstrap-MLM_PROXY.sh
+----
+
++
+
+Replace [literal]``MLM_PROXY`` with the host name of your proxy.
+   With the [option]``--script`` option you save the bootstrap script with a descriptive name.
+   If needed, use additional command line options to customize your bootstrap script.
+   To view available options type [command]``mgr-bootstrap --help`` from the command line.
+
+.. OPTIONAL: Edit the resulting bootstrap script.
+--
+
+. Execute the bootstrap script.
+  There are two options:
+
++
+
+* Transfer the script to the client and run it directly on the clients.
+  Alternatively, because the proxy forwards requests to the server, the client can download it from http://MLM_PROXY/pub/bootstrap.sh.
```

Comment:
As the example now modifies the script name, use that in this example as well.

```suggestion
  Alternatively, because the proxy forwards requests to the server, the client can download it from http://MLM_PROXY/pub/bootstrap-MLM_PROXY.sh.
```